src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160831_attribute_namespace.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="attribute-namespace-1">
+        <sql stripComments="true">
+            CREATE TABLE ATTRIBUTE_NAMESPACE (
+                id bigint(20) unsigned AUTO_INCREMENT,
+                name varchar(254) NOT NULL,
+                PRIMARY KEY (id)
+            );
```

Comment:
my opinion here is that namespace should be a column on attribute rather than a table of its own.

i can imagine a world where namespaces get e.g. permissions and have extra data associated with them. so i'm willing to entertain the idea of a view that makes attribute tables look like they have the namespace column jammed on. that makes the code nicer at least.